### PR TITLE
Revamp that_was_easy

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: workshops/that_was_easy/README.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # That Was Easy Workshop
 
+**We recommend going through this workshop in Google Chrome.**_
```
Comment (zachlatta) [inline workshops/that_was_easy/README.md]: Is that `_` at the end of the line intentional?

File: workshops/that_was_easy/README.md [hunk 2]
```diff
@@ -27,30 +34,30 @@ Much like in Personal Website workshop, we'll be using HTML to create stuff visi
...
+1. First log into your [Cloud9](https://c9.io) account and go to the dashboard. Open workspace by clicking "Open" in our dashboard.
 
    ![](../img/c9_dashboard.png)
 
 2. Next, we'll create a new folder for this workshop. Right-click your `projects` folder, select "New Folder," and name it `that_was_easy`.
 3. In this folder, right-click again, select "New File," and name the new file `index.html`. This is where we will be writing HTML to actually create the button.
 4. Create another file in the `that_was_easy` folder, and name this one `main.js`. This is where we will be writing our JavaScript to make the button play sounds.
 5. Create another file in the `that_was_easy` folder, and name this one `styles.css`. This is where we will be making our button look pretty.
-6. We've provided an amusing sound for you [here](sounds/that_was_easy.mp3). If you click on the filename, and then click `View Raw`, you should be at a page with an audio player. Right-click that page and choose `Save As`.
+6. We've provided an amusing sound for you -- download the file by clicking [here](sounds/that_was_easy.mp3).
```
Comment (zachlatta) [inline workshops/that_was_easy/README.md]: I think club members will struggle to download the file. We should have a GIF.

Comment (MaxWofford) [inline workshops/that_was_easy/README.md]: @zachlatta The link downloads the mp3 directly when using lecture-hall.

**Comment by MaxWofford** [comment]
@zachlatta Ping on this PR

File: workshops/that_was_easy/README.md (hunk 2 above)
Comment (zachlatta) [inline workshops/that_was_easy/README.md]: Woah.

**Pull request merged**